src/inc/stackstring.h

```diff
@@ -0,0 +1,196 @@
+// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license. 
+// See the LICENSE file in the project root for more information. 
+
+
+#ifndef __STACKSTRING_H_
```

Comment:
@janvorli : I did not mean to use the Wrapper Apis in PAL. Ideally the Wrapper can become a PAL Api, but i am not gunning for it yet.  
The wrappers which take path as an out parameter will need to still work with PAL.
The ones which don't can bypass the Wrappers to PAL

@gkhanna79 : The errors i get is by including "sstring.h" into "longpathnamewrappers.h". If i remove it and the call sites already have access to the type, it works [ This seems to be the case all the time]. I guess i was eagerly including headers into longpathnamewrappers so that all the types it needs are present.

I will remove the usage of PathWCharString from VM now and stick with SString